src/Web/Grand.Web/Views/PdfTemplates/OrderPdfTemplate.cshtml

```diff
@@ -87,6 +88,7 @@
                                 string paymentMethodStr = paymentMethod != null ? paymentMethod.FriendlyName : order.PaymentMethodSystemName;
                             }
                             <li>@(string.Format(_translationService.GetResource("PDFInvoice.PaymentMethod", order.CustomerLanguageId), paymentMethodStr))</li>
+                            @await Component.InvokeAsync("Widget", new { widgetZone = "payment_method_below", additionalData = @order.Id })
```

Comment:
Please change the name for the widget zone, better name will be i.e: pdf_payment_method_below

Reply:
@KrzysztofPajak updated the zone name